ui/tests/integration/components/list-test.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'vault/tests/helpers';
+import { render, click, find, findAll } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+import { v4 as uuidv4 } from 'uuid';
+import sinon from 'sinon';
+import { setupMirage } from 'ember-cli-mirage/test-support';
+import { overrideResponse } from 'vault/tests/helpers/stubs';
+import { clickTrigger } from 'ember-power-select/test-support/helpers';
+
+import { createSecretsEngine } from 'vault/tests/helpers/secret-engine/secret-engine-helpers';
+import { SECRET_ENGINE_SELECTORS as SES } from 'vault/tests/helpers/secret-engine/secret-engine-selectors';
+import { GENERAL } from 'vault/tests/helpers/general-selectors';
+
+module('Integration | Component | secret-engine/list', function (hooks) {
+  setupRenderingTest(hooks);
+  setupMirage(hooks);
+
+  hooks.beforeEach(function () {
+    this.server.post('/sys/capabilities-self', () => ({
+      data: {
+        capabilities: ['root'],
+      },
+    }));
+    this.store = this.owner.lookup('service:store');
+    this.version = this.owner.lookup('service:version');
+    this.flashMessages = this.owner.lookup('service:flash-messages');
+    this.flashMessages.registerTypes(['success', 'danger']);
+    this.flashSuccessSpy = sinon.spy(this.flashMessages, 'success');
+    this.flashDangerSpy = sinon.spy(this.flashMessages, 'danger');
+    this.uid = uuidv4();
+    // generate a model of cubbyhole, kv2, and nomad
+    this.secretEngineModels = [
+      createSecretsEngine(this.store, 'cubbyhole', 'cubbyhole-test'),
+      createSecretsEngine(this.store, 'kv', 'kv2-test'),
+      createSecretsEngine(this.store, 'aws', 'aws-1'),
+      createSecretsEngine(this.store, 'aws', 'aws-2'),
+      createSecretsEngine(this.store, 'nomad', 'nomad-test'),
+    ];
+  });
+
+  test('it allows you to disable an engine', async function (assert) {
+    const enginePath = 'kv2-test';
+    this.server.delete(`sys/mounts/${enginePath}`, () => {
+      assert.ok(true, 'Destroy record is called and deletes the engine');
```

Comment:
we're staying away from the use of assert.ok. 

assert.ok | Asserts that a value is truthy | General truthiness check | assert.ok(value);
assert.true | Asserts that a value is true | Precise comparison to boolean true | assert.true(value);

```suggestion
      assert.true(true, 'Destroy record is called and deletes the engine');
```